README.md

```diff
@@ -4,11 +4,21 @@
 [Here](https://arxiv.org/abs/1911.05567) is the paper describing the project and experiments in detail.
 
 ## Package
-The DARTS package can be installed using:
+* The DARTS package can be installed as follows:
 ```
 pip install DARTSeg
 ```
-Download the pretrained model from [here](https://drive.google.com/file/d/1-reUDvwBhSOUqOa48W9Vgh_LN3F5ZRjQ/view?usp=sharing) and follow the steps to perform segmentation
+
+Upon seccessful installation, you should be able to do `perform_pred.py --help`.
+
+* Download the pretrained models from [here](https://drive.google.com/file/d/1-reUDvwBhSOUqOa48W9Vgh_LN3F5ZRjQ/view?usp=sharing) as follows:
+
+```
+gdown https://drive.google.com/uc?id=1-reUDvwBhSOUqOa48W9Vgh_LN3F5ZRjQ -O saved_model_wts.zip
+unzip saved_model_wts
+```
+
```

Comment:
Downloading the models like above should be a better practice given most people would want to do it from their terminal rather than having to go to Google Drive web interface and click download etc.